dataset/dictionary/y-events.json5

```
@@ -242,72 +249,83 @@
   //   tags: [ "event", "object" ],
   // },
   {
-    en: "water cannon",
+    en: "Water Cannon",
```

According to [v3.8 preview page](https://act.hoyoverse.com/ys/event/e20230624preview/index.html?game_biz=hk4e_global&hyl_presentation_style=fullscreen&hyl_landscape=true&hyl_hide_status_bar=true), w and c are lowercase. (See Secret Supper Paradise page > Park Attractions > Spino Blaster)

```suggestion
    en: "water cannon",
```

However, if there are any other official sources written as "Water Cannon" (with uppercase), uppercase may be preferrable. Did you see such sources?

---

i will change it back